index.html

```diff
@@ -40,6 +40,7 @@ <h1>Luxury </h1>
                 <a class="btn lx" href="LUxury.Html">Learn More</a>
 
             </div>
```

Comment:
Could you close all spaces in the nested element in your codebase? Additionally I want to point you to some observations on your designs, the alignments, you have to use flexbox to center your containers, so you can use CSS flexbox rule of justify-content: center and align-items: center:
![image](https://github.com/Elilly17/projectfrontend/assets/87186552/67e1df34-a6ab-4725-b07b-ea674ea8ef02)
![image](https://github.com/Elilly17/projectfrontend/assets/87186552/17bf1955-ca62-4206-b20b-f689b822dd2c)
![image](https://github.com/Elilly17/projectfrontend/assets/87186552/9e38e9c9-7e2b-4c6d-b75b-181cca841e6a)
![image](https://github.com/Elilly17/projectfrontend/assets/87186552/e0d8295f-8210-4b30-ad15-8bff377f15ac)
another thing is that in the learn more button, you can use an html link tag element property called target and set it to _blank, this will allow a new page to be rendered on a new tap on the browser and leaving the previous page opened, that is a good user experience.

I will like you to pay attention to the following;
1. In your codebase, I will suggest you pay attention to the file namimg conventions, they should all begin with small letters and end with small letters, do not use upper case for file names.
2. The image on the index page is broken, maybe you did not import the image to your project folder. kindly import the image and select the right path, and the image will display.
3. This image url 
![image](https://github.com/Elilly17/projectfrontend/assets/87186552/30c298e2-7d45-453e-84c0-8db459a67da1)
is too long and it makes your code base untidied, if you want to use a url for your image, shorten the url or better still, download the image and use it directly in your codebase.
4. Your file extention too (i.e .html) has to be lowercase .

Work on these cahneges and if you have any questions, do not hesitate to ask.